### PR TITLE
Issue#2483: enable copy help texts csr method of certificate creation

### PR DESCRIPTION
**Pull request by LeonamMoura**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)
dojot/dojot#2483

* **Other information**:

### CODE REVIEW

**Review by LuanEdCosta — COMMENTED** [review]
@LeonamMoura Maybe you can add the copy to clipboard button that you created some days ago to allow the user to copy the command. What do you think?

File: src/js/views/createCertificate/translations/en.createCertificate.i18n.json [hunk 1]
```diff
@@ -14,7 +14,8 @@
     "generateCsrHelp": "How to Generate a CSR?",
     "csrHelpSteps": {
       "step1": "Step 1: ",
-      "step1Text": "Run the command: openssl req  -newkey 'rsa:2048'  -nodes  -sha256  -keyout 'privatekey.pem'  -out 'csr.pem'  -subj '/CN=Any value'",
+      "runTheCommand": "Run the command:",
+      "generateCsrCommand": "openssl req  -newkey 'rsa:2048'  -nodes  -sha256  -keyout 'privatekey.pem'  -out 'csr.pem'  -subj '/CN=Any value'",
```
Comment (LuanEdCosta) [inline src/js/views/createCertificate/translations/en.createCertificate.i18n.json:18]: You can remove the extra white space between command options

File: src/js/views/createCertificate/translations/pt_br.createCertificate.i18n.json [hunk 2]
```diff
@@ -14,7 +14,8 @@
     "generateCsrHelp": "Como Gerar um CSR?",
     "csrHelpSteps": {
       "step1": "Passo 1: ",
-      "step1Text": "Execute o comando: openssl req  -newkey 'rsa:2048'  -nodes  -sha256  -keyout 'privatekey.pem'  -out 'csr.pem'  -subj '/CN=Any value'",
+      "runTheCommand": "Execute o comando",
+      "generateCsrCommand": "openssl req  -newkey 'rsa:2048'  -nodes  -sha256  -keyout 'privatekey.pem'  -out 'csr.pem'  -subj '/CN=Any value'",
```
Comment (LuanEdCosta) [inline src/js/views/createCertificate/translations/pt_br.createCertificate.i18n.json:18]: You can remove the extra white space between command options

File: src/js/views/createCertificate/layout/style.js [hunk 3]
```diff
@@ -18,4 +23,9 @@ export default makeStyles(theme => ({
       minWidth: '16rem',
     },
   },
+  inputCommand: {
+    fontSize: 14,
+    marginLeft: theme.spacing(1),
```
Comment (LuanEdCosta) [inline src/js/views/createCertificate/layout/style.js:28]: I think that a `marginRight` would be great and maybe you can set a fixed width for the input.

File: src/js/views/createCertificate/translations/en.createCertificate.i18n.json [hunk 4]
```diff
@@ -14,7 +14,8 @@
     "generateCsrHelp": "How to Generate a CSR?",
     "csrHelpSteps": {
       "step1": "Step 1: ",
```
Comment (LuanEdCosta) [inline src/js/views/createCertificate/translations/en.createCertificate.i18n.json:16]: The text has a white space, but for some reason it is not shown on screen.

image

File: src/js/views/createCertificate/translations/pt_br.createCertificate.i18n.json [hunk 5]
```diff
@@ -14,7 +14,8 @@
     "generateCsrHelp": "Como Gerar um CSR?",
     "csrHelpSteps": {
       "step1": "Passo 1: ",
-      "step1Text": "Execute o comando: openssl req  -newkey 'rsa:2048'  -nodes  -sha256  -keyout 'privatekey.pem'  -out 'csr.pem'  -subj '/CN=Any value'",
+      "runTheCommand": "Execute o comando",
```
Comment (LuanEdCosta) [inline src/js/views/createCertificate/translations/pt_br.createCertificate.i18n.json:17]: Missing `:`

The english translation has `:`
image

**Review by LuanEdCosta — CHANGES_REQUESTED** [review]

**Review by LuanEdCosta — APPROVED** [review]

**Review by luisfelipefurlan — APPROVED** [review]

**Pull request merged**